di_website/static/css/di_website.css

```diff
@@ -18,3 +18,12 @@
     width: 100%;
     height: 100%;
 }
+
+.resources_filter_button {
```

Comment:
Follow FF convention on the class names. I'd suggest `resources-filter--button`

Reply:
Updated the class